Assets/Scripts/DataController.cs

```diff
@@ -138,7 +145,77 @@ void Start()
                 }
             }
         }
+
+        if (markerPrefab != null)
+        {
+            AddMarker("NCP", 0f, 90f, localSiderialTime, colorOrange);
+            AddMarker("SCP", 0f, -90f, localSiderialTime, colorOrange);
+            AddMarker("VE", 0f, 0f, localSiderialTime, colorGreen);
+            AddCircumferenceMarker("equator", colorBlue, markerLineWidth);
+            AddLineMarker("poleLine", colorOrange, GameObject.Find("NCP"), GameObject.Find("SCP"), markerLineWidth);
+        }
     }
+
+    void AddMarker(string name, float RA, float dec, double lst, Color color)
+    {
+        Marker marker = new Marker(name, RA, dec);
+        GameObject markerObject = Instantiate(markerPrefab, this.transform.position, Quaternion.identity);
+        markerObject.transform.parent = allMarkers.transform;
+        MarkerComponent newMarker = markerObject.GetComponent<MarkerComponent>();
+        newMarker.label.text = name;
+        newMarker.markerData = marker;
+        markerObject.name = name;
+        changeStarColor(markerObject, color);
+        if (showHorizonView)
+        {
+            markerObject.transform.position = newMarker.markerData.CalculateHorizonPosition(radius, lst, 0);
+        }
+        else
+        {
+            markerObject.transform.position = newMarker.markerData.CalculateEquitorialPosition(radius);
+        }
+    }
+
+    void AddCircumferenceMarker(string name, Color color, float lineWidth)
+    {
+        GameObject circumferenceObject = new GameObject();
+        circumferenceObject.name = name;
+        circumferenceObject.transform.parent = allMarkers.transform;
+        int segments = 360;
+        LineRenderer lineRendererCircle = circumferenceObject.AddComponent<LineRenderer>();
+        lineRendererCircle.useWorldSpace = false;
+        lineRendererCircle.startWidth = lineWidth;
+        lineRendererCircle.endWidth = lineWidth;
+        lineRendererCircle.positionCount = segments + 1;
+        lineRendererCircle.material = markerMaterial;
+        lineRendererCircle.material.color = color;
+
+        int pointCount = segments + 1; // add extra point to make startpoint and endpoint the same to close the circle
+        Vector3[] points = new Vector3[pointCount];
+
+        for (int i = 0; i < pointCount; i++)
+        {
+            float rad = Mathf.Deg2Rad * (i * 360f / segments);
+            points[i] = new Vector3(Mathf.Sin(rad) * radius, 0, Mathf.Cos(rad) * radius);
+        }
+
+        lineRendererCircle.SetPositions(points);
+    }
+
+    void AddLineMarker(string name, Color color, GameObject go1, GameObject go2, float lineWidth)
```

Comment:
Change name to markerName

Reply:
Sounds good.

Reply:
Same as above, in this case I am trying to set the Unity name property of the game object.

Reply:
Changed to markerName.